maestro-cli/src/main/java/maestro/cli/command/CloudCommand.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ *
+ *  Copyright (c) 2022 mobile.dev inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *
+ */
+
+package maestro.cli.command
+
+import maestro.cli.api.ApiClient
+import maestro.cli.cloud.CloudInteractor
+import picocli.CommandLine
+import picocli.CommandLine.Option
+import java.io.File
+import java.util.concurrent.Callable
+
+@CommandLine.Command(
+    name = "cloud",
```

Comment:
Add a description for the command. Should be pretty detailed since this is how a lot of folks will learn about the command. Maybe we can link to docs as well